src/orquestra/quantum/operators/_pauli_operators.py

```diff
@@ -542,6 +542,9 @@ def __repr__(self):
             return str(zero_identity_term)
         return " + ".join([str(term) for term in self.terms])
 
+    def __hash__(self):
+        return hash(tuple(self.terms))
```

Comment:
[praise] Ha! Smart!

Reply:
Yeah, I'm going to need this later for a qulacs PR that depends on it.